contracts/cw20-staked-balance-voting/src/contract.rs

```diff
@@ -226,6 +232,36 @@ pub fn query_dao(deps: Deps) -> StdResult<Binary> {
     to_binary(&dao)
 }
 
+pub fn query_is_active(deps: Deps) -> StdResult<Binary> {
+    let threshold = ACTIVE_THRESHOLD.may_load(deps.storage)?;
+    match threshold {
+        Some(threshold) => {
+            let token_contract = TOKEN.load(deps.storage)?;
+            let staking_contract = STAKING_CONTRACT.load(deps.storage)?;
+            let total_potential_power: TokenInfoResponse = deps
+                .querier
+                .query_wasm_smart(token_contract, &cw20_base::msg::QueryMsg::TokenInfo {})?;
```

Comment:
Seems like we don't need this in the `AbsoluteCount` case. How about moving that into the match statement below?